modules/serverless-rn-1-16-0.adoc

```diff
@@ -15,6 +15,8 @@
 [id="known-issues-1-16-0_{context}"]
 == Known issues
 
+* You must upgrade {product-title} to version 4.6.30, 4.7.11, or higher before upgrading to {ServerlessProductName} 1.16.0.
```

Comment:
upgrading to `1.16.0` or `1.17.0` ?

Reply:
1.16.0 - this is in the `serverless-rn-1-16-0.adoc` file, i.e. the 1.16.0 release notes, but I thought I'd kill two birds with one PR.